### DISCUSSION TITLE
Pause server during map updates

### DISCUSSION THREAD

**Discussion by GrandpaGameHacker** [Ismoh/NoitaMP#70] [Contribution - How can you help us?]
in todo ```pause game somehow, until all clients sent "map received" to server```
PauseSimulation is at 0x00FEE41A, this stops the game from advancing the physics/input/particles/etc.
it is a boolean flag, so you have to write 1 to this address for it to pause and zero to unpause.
I am unsure if this will work but I'm hoping it does.

**Comment by Ismoh** [comment]
Thanks a ton for your help!
I think this is more or less the same as pressing esc and opening the menu?
This is awesome, because before pausing the game, I am able to show a hint as ui text, being like "Game paused until client downloaded the map!"
And in addition it won't lead to desync or server needs to sync world while playing, which would lead to lag!
Thanks again ❤️

**Reply by GrandpaGameHacker** [reply]
Nw Ismoh ❤️ 💯

**Comment by Ismoh** [comment]
Looks like this isn't necessary onymore, because of some rework.
But I really appreciate your help and thoughts on NoitaMP!

Going to lock this discussion to keep things organised.
